package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@datadog/rum-react-integration",
   "version": "1.0.0-alpha",
-  "private": "true",
```

Comment:
I don't think we should remove this, as we have no intentions of publishing it in public npm repos.